src/components/Button/Button.styles.js

```diff
@@ -5,7 +5,7 @@ import { type } from './constants';
 const styles = {
   inner: css({
     display: 'block',
-    marginBottom: '1px',
+    marginBottom: '2px',
```

Comment:
Maybe we should add this to the `Space` util?

Reply:
My thinking is that this is a one-off value that shouldn't be used elsewhere in the app. How about a constant that lives in this file?